### ISSUE TITLE
New block: Conversion Trigger

### ISSUE THREAD

**Issue by Truthans** [Blueshoe/website#154]
Hey @MaksymOsovitnii,

we'd like to build a new content block which has the purpose of converting visitors to leads.

Please have a look at the following figma file:
https://www.figma.com/file/Ut1rNIDCh1YjlwbwbHSiWr/Blueshoe_page_element?type=design&node-id=18%3A606&mode=dev&t=E6qhqRGq1DOCXnEZ-1

I want to be able to adapt the following things:
- Change headlines and subheadlines
- Exchange the icons
- Change the foto image at the bottom

For mobile, I'd like to use the design with the fade out on the right.

Can you provide an estimation for this to be implemented?

**Comment by MaksymOsovitnii** [comment]
@Truthans , it will take about 8 hours to implement, test and create a histoire for this component.

**Comment by Truthans** [comment]
@MaksymOsovitnii alright, then you can go ahead and build the thing, if you don't have any more questions. :)

**Comment by MaksymOsovitnii** [comment]
@Truthans , sure, will start working on it on Wed, ok?

**Comment by MaksymOsovitnii** [comment]
@Truthans 
Hi Hannes, I have several questions in terms of the new block I working on 

1. Titles in the new block are bolder that other titles across the website, should I stick to figma?
2. Green subtitle is not in uppercase as other green subtitles across the website, should I stick to figma?
3. How should the slider look like on mobile device when the 2nd slide is active? do we see the part of the previous slide because of the slide form? 
4. The green slide is the hovered one or this is just a separate type of slide with background color and text in the center?

**Comment by Truthans** [comment]
> @Truthans Hi Hannes, I have several questions in terms of the new block I working on
> 
> 1. Titles in the new block are bolder that other titles across the website, should I stick to figma?

Yes, in this case let's stick to Figma. Since this is supposed to break out a little bit of the normal style, it's fine to be a little different.

> 2. Green subtitle is not in uppercase as other green subtitles across the website, should I stick to figma?

That should indeed be in all Caps.

> 3. How should the slider look like on mobile device when the 2nd slide is active? do we see the part of the previous slide because of the slide form?

Good question. I would say so, that you can still see the pointy part from the first slide. Potentially adding a fade effect on the left side too, to make it consistent?

> 4. The green slide is the hovered one or this is just a separate type of slide with background color and text in the center?

It's just a separate type of slide with background colour. No hover effects at the moment.

**Comment by MaksymOsovitnii** [comment]
@Truthans , thanks, will be back with the demo.
